migrations/20220226194503-add_user_type_column.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const TABLE_NAME="user";
+module.exports = {
+  async up (queryInterface, Sequelize) {
+     await queryInterface.addColumn(TABLE_NAME, 'userTypeId', {
+       type: Sequelize.DataTypes.INTEGER(11),
+       defaultValue: 1
+     }).then(() => queryInterface.addConstraint(TABLE_NAME, {
+      fields: ['userTypeId'],
+      type: 'foreign key',
+      name: 'custom_fkey_user_type_id',
+      references: { 
+        table: 'userTypes',
+        field: 'id'
+      },
+      onDelete: 'no action',
+      onUpdate: 'no action'
```

Comment:
We should add cascading options here, for when the values are deleted.updated on parent table, they should be dealt the same way here